textures/lores/haumea-rings.png.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2019 Askaniy
```

Comment:
2018 actually

Reply:
thanks, updated

Reply:
should we use full names where possible?

Reply:
updated